supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -325,12 +328,25 @@ trait SharedFileSystemAsyncJobExecutionActor
     }
   }
 
+  //TODO: This maybe should be a config value
+  /** Number of miliseconds between isAlive calls */
+  val isAliveTimeout = 20000
+
+  private case class IsAliveCash(last: Calendar, cash: Boolean)
+  private var isAliveCash: mutable.Map[SharedFileSystemJob, IsAliveCash] = mutable.Map()
+
   def isAlive(job: SharedFileSystemJob): Boolean = {
-    val argv = checkAliveArgs(job).argv
-    val stdout = pathPlusSuffix(jobPaths.stdout, "check")
-    val stderr = pathPlusSuffix(jobPaths.stderr, "check")
-    val checkAlive = new ProcessRunner(argv, stdout, stderr)
-    checkAlive.run() == 0
+    val now = Calendar.getInstance()
+    jobLogger.debug(s"executing isAlive for ${job.jobId}")
+    if (isAliveCash(job).cash && (isAliveCash(job).last.getTimeInMillis + isAliveTimeout) < now.getTimeInMillis) {
+      val argv = checkAliveArgs(job).argv
+      jobLogger.debug(s"Executing shell for ${job.jobId}: " + argv.mkString("'", " ", "'"))
```

Comment:
While it does work on SGE I see this in my log:
`
[2016-08-29 13:37:07,74] [debug] SharedFileSystemAsyncJobExecutionActor [13f02c3apipe.fastqsync:NA:1]: Executing shell for 28265: 'ps 28265'
`

I would expect here to see `qstat -j 28265`, he must also execute this because it is working. Anyone an idea what goes wrong here?